Add support to search after (#198)
logstash-plugins/logstash-input-elasticsearch: docs/index.asciidoc

request and initiates the scrolling process. The timeout applies per
round trip (i.e. between the previous scroll request, to the next).

[id="plugins-{type}s-{plugin}-seearch_api"]
===== `search_api`

* Value can be any of: `auto`, `search_after`, `scroll`
* Default value is `auto`

> `auto` uses `search_after` for Elasticsearch version `8.0.0` or higher, otherwise, fall back to `scroll` API.

Reviewer: ```suggestion
With `auto` the plugin uses the `search_after` parameter for Elasticsearch version `8.0.0` or higher, otherwise, the `scroll` API is used instead.
```

`search_after` uses {ref}/point-in-time-api.html#point-in-time-api[point in time] and sort value to search.

> The query requires to have at least one `sort` field. Check out <<plugins-{type}s-{plugin}-query>>.

Reviewer: ```suggestion
The query requires at least one `sort` field, as described in the <<plugins-{type}s-{plugin}-query>> parameter.
```
